include/core/simulation_control.h

```diff
@@ -499,19 +492,41 @@ class SimulationControlTransient : public SimulationControl
   // Max time step
   double max_dt;
 
+  // Time last output
+  double time_last_output;
+
+  // Specific output frequency for time output control
+  double output_time_frequency;
+
+  // Specific output times for time output control
+  std::vector<double> output_times_vector;
+
+  // Counter to move between output times given in previous vector
+  unsigned int output_times_counter = 0;
+
+  // Variable to check whether we still have specific times to check in output
+  // times vector
+  bool no_more_output_times = false;
```

Comment:
Are you sure that it's a good idea to declare default value here instead of in the constructor?